tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand_001_pos.ksh

```diff
@@ -27,6 +27,7 @@
 
```

Comment:
Since we are not going to run 001 test, can we just leave this file as-is? The auto-expand zed functionality probably won't work with files.